src/generate/rust.rs

```diff
@@ -852,7 +852,7 @@ where
                     input,
                     #code_label,
                     #parse_node_kind,
-                ).map(handle).map_err(|err| err.map_partial(|x| handle(x).source_info()))
+                ).map(handle)
```

Comment:
Oh, can you inline the `handle` closure?

Reply:
So
```rust
gll::runtime::Parser::parse(
    input,
    #code_label,
    #parse_node_kind,
).map(|forest_and_node| OwnedHandle {
    forest_and_node,
    _marker: PhantomData,
})
```
?

(This looks like a use case for a `OwnedHandle::new`, actually)